src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -83,6 +102,9 @@ public LocalCall(String functionName, Optional<List<?>> arg,
         arg.ifPresent(arg -> payload.put("arg", arg));
         kwarg.ifPresent(kwarg -> payload.put("kwarg", kwarg));
         metadata.ifPresent(m -> payload.put("metadata", m));
+        timeout.ifPresent(timeout -> payload.put("timeout", timeout));
```

Comment:
if it will be fixed in salt then its ok. Only in case it will or cant be fixed in salt we should make sure in the library that you cant end up configuring something that will never work the way you expect.